ledger/src/blockstore.rs

```diff
@@ -170,6 +174,50 @@ impl<T> AsRef<T> for WorkingEntry<T> {
     }
 }
 
+#[derive(Clone, Copy, PartialEq, Eq, Debug)]
+pub struct LastFECSetCheckResults {
+    block_id: Option<Hash>,
+    is_retransmitter_signed: bool,
+}
+
+impl LastFECSetCheckResults {
+    pub fn report_metrics(&self, slot: Slot, bank_hash: Hash, cluster_type: ClusterType) {
+        if self.block_id.is_none() {
+            datapoint_warn!(
+                "incomplete_final_fec_set",
+                ("slot", slot, i64),
+                ("bank_hash", bank_hash.to_string(), String)
+            );
```

Comment:
Just to confirm, this is not currently spamming metrics server because last FEC set is already 32+ data shreds on mainnet, right?